shell/browser/file_system_access/file_system_access_permission_context.cc

```diff
@@ -423,6 +426,7 @@ struct FileSystemAccessPermissionContext::OriginState {
   // PermissionGrantDestroyed().
   std::map<base::FilePath, PermissionGrantImpl*> read_grants;
   std::map<base::FilePath, PermissionGrantImpl*> write_grants;
+  std::unique_ptr<base::RetainingOneShotTimer> cleanup_timer;
```

Comment:
_(nit)_ add a blank line above this so that it's clear at-a-glance that comment at 424-426 doesn't apply to `cleanup_timer`